src/gradleinttest.sh

```diff
@@ -37,13 +37,10 @@ gradleinttest () {
     echo "Running ${test_to_run} test with data project env var unset: ${google_data_project}"
   fi
 
-  if [[ -n "${google_project}" ]] && [ -f jade-dev-account.json ] && [ -f jade-dev-account.pem ] && [[ "${test_to_run}" != "" ]]; then
+  if [[ -n "${google_project}" ]] && [ -f ${GOOGLE_APPLICATION_CREDENTIALS} ] && [[ "${test_to_run}" != "" ]]; then
```

Comment:
How come you removed the existence check on the PEM file here? Just curious.

Reply:
Oh, I should add that back! I'll check that GOOGLE_SA_CERT exists.